tests/rados/test_bluestoretool_workflows.py

```diff
@@ -283,7 +283,6 @@ def run(ceph_cluster, **kw):
         for osd_service in osd_services:
             cephadm.shell(args=[f"ceph orch restart {osd_service}"])
         time.sleep(30)
-        assert rados_obj.run_pool_sanity_check()
```

Comment:
The module `run_pool_sanity_check` is supposed to ensure cluster does not have any health warning after the test workflow execution.
Recently, it was observed that if the cluster is having an active Cluster health warning encountered during any of the proceeding tests, the `bluestoretool` test also fails even though all the commands in the `bluestoretool_workflow` had passed, basically resulting in a false failure.